numpy/polynomial/_polybase.py

```diff
@@ -123,6 +123,10 @@ def _line(self):
         pass
 
     @abstractmethod
+    def _interp(self):
+        pass
```

Comment:
In fact, adding abstract methods is an API change, so you couldn't even do that if you did implement all the others, in case someone out there is subclassing `PolynomialBase`